src/math/bspline.hpp

```diff
@@ -11,6 +11,16 @@ namespace precice::math::bspline {
  * @pre splineDegree must be larger than 0.
  * @pre t must be within [ts.min; ts.max].
  */
-Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t);
+class bspline {
```

Comment:
```suggestion
class Bspline {
```
Because it's a class.